.devcontainer/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apt-get update && apt-get install dos2unix git python2 curl -y \
     && cd /features \
     # locking to the latest master commit in this repo to prevent breaking changes
     # We should update this commit hash from time to time to
-    && git checkout 1869e5931cfe0517f75d58cb70863a6b4874c487
+    && git checkout 96bff0097028001e6e4126c5528d37cb8c13e785
```

Comment:
where did this magic value come from? and can it be computed instead of magic. or document we you get it from.

Reply:
This is the latest commit of this repo:
https://github.com/devcontainers/features

Basically, we shouldn't install the Dev Container features directly from this repo. The way to install features is to add this in the configuration file:
```
"features": {
    "ghcr.io/devcontainers/features/go:1": {
        "version": "1.18"
    },
    "ghcr.io/devcontainers/features/docker-in-docker:1": {
        "version": "latest",
        "moby": true
    }
```

However, GP blocks it, so our workaround is to install the features directly from the repo in the Dockerfile. We set to the latest commit as we had a case which this repo had breaking changes which we did not caught.

Reply:
@ilaner  why not ask GP to approve this URL?